src/api/exporters/ezsLocals/linkDataset.js

```diff
@@ -1,18 +1,52 @@
+import validUrl from 'valid-url';
+import { set } from 'lodash';
+import deepObject from 'deep-object-map';
+
+/**
+ * replace uri blankNode to unique uri
+ * @param {object} data
```

Comment:
Maybe describe the structure of the `data` parameter.
Is it JSON-LD?
An intermediary form?